Libraries/LibJS/Lexer.cpp

```diff
@@ -30,8 +30,27 @@
 #include <ctype.h>
 #include <stdio.h>
 
+//#define LEXER_DEBUG
+
 namespace JS {
 
+#define TWO_CHARS_VIEW()                                           \
+    StringView view;                                               \
+    do {                                                           \
+        char second_char = m_source[m_position];                   \
+        char two_chars[] { (char)m_current_char, second_char, 0 }; \
+        view = { two_chars };                                      \
+    } while (0)
+
+#define THREE_CHARS_VIEW()                                                       \
+    StringView view;                                                             \
+    do {                                                                         \
+        char second_char = m_source[m_position];                                 \
+        char third_char = m_source[m_position + 1];                              \
+        char three_chars[] { (char)m_current_char, second_char, third_char, 0 }; \
+        view = { three_chars };                                                  \
+    } while (0)
```

Comment:
This is what I asked about on IRC this morning - made it two macros instead, I guess that works?!

Reply:
Any particular reason why you can't just use `m_source`?
`StringView view { &m_source[m_position-1], 2 }` for the two chars one?
(What even is `m_source`? a String?)

Reply:
> Any particular reason why you can't just use m_source?

'cause I didn't think of that, that's why :smile:

> (What even is m_source? a String?)

A StringView :)

I'll try this - thanks!

Reply:
that totally worked lol. To not see the forest for the trees, or something... ty!